docs/source/release_notes.rst

```diff
@@ -1,17 +1,63 @@
 .. ...........................................................................
-.. © Copyright IBM Corporation 2020, 2021                                          .
+.. © Copyright IBM Corporation 2020, 2021, 2023                              .
 .. ...........................................................................
 
 ========
 Releases
 ========
-
-Version 1.5.0-beta.1
+Version 1.6.0-beta.1
 ====================
 
 New Modules
 -----------
 
+- ``zos_volume_init`` - Can initialize volumes or minidisks on target z/OS systems which includes creating a volume label and an entry into the volume table of contents (VTOC).
+
+Minor Changes
+-------------
+
+- ``zos_blockinfile`` - Adds an enhancement to allow double quotes within a block.
+- ``zos_copy`` - Adds an enhancement to keep track of modified members in a destination dataset, restoring them to their previous state in case of a failure.
+- ``zos_data_set`` - Adds a new option named force to enable deletion of a data member in a PDSE that is simultaneously in use by others.
+- ``zos_job_query`` - Enables embedded positional wild card placement throughout `job_name` and `job_id` parameters.
```

Comment:
I don't think RST does anything with single single  back ticks "`" , if you double then `` you will get colorized text might be nice for the module properties, we did this in 1.4.0 doc as an example. Some other options here in the release notes might benefit.